arian1917/arian1917/Program.cs

```diff
@@ -0,0 +1,579 @@
+﻿using System.Data;
+using System.Diagnostics.Metrics;
+using System.Globalization;
+using System.Linq.Expressions;
+using System.Numerics;
+using System.Reflection.Metadata.Ecma335;
+using System.Runtime.CompilerServices;
+using Microsoft.Data.Sqlite; 
+
+
+namespace habitTracker{
+    class Program{
+        
+        static string connectionString = @"Data Source=habit_tracker.db"; 
+        
+        static void Main(string[] args)
+        {
+            InitializeDataBase(); 
+            HabitSelection(); 
+        }
+
+        private static void InitializeDataBase()
+        {
+            using (var connection = new SqliteConnection(connectionString))
```

Comment:
Use the simpler "Using Declaration" syntax to save an indentation level against the "Using Statement" syntax:
```
// Using Statement
using (var connection = new SqliteConnection(connectionString))
{
    // Code here.
}
```
vs
```
// Using Declaration
using var connection = new SqliteConnection(connectionString);
// Code here.
```